crates/winch/src/abi/mod.rs

```diff
@@ -0,0 +1,151 @@
+use crate::isa::reg::Reg;
+use smallvec::SmallVec;
+use std::ops::{Add, BitAnd, Not, Sub};
+use wasmtime_environ::{WasmFuncType, WasmType};
+
+pub(crate) mod addressing_mode;
+pub(crate) mod local;
+
+/// Trait implemented by a specific ISA and used to provide
+/// information about alignment, parameter passing, usage of
+/// specific registers, etc
```

Comment:
I'll note here but applies generally: overall in doc-comments we've tried to stick to capitalized-with-periods-at-end style; many comments here don't end with a `.`.

Reply:
Done, thanks! Sorry for not noticing this earlier!